experiments/rtp-hdrext/playout-delay/index.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: default
+title: playout-delay
+permalink: /experiments/rtp-hdrext/playout-delay/
+---
+
+Name: "Playout Delay" ; "RTP Header Extension to control Playout Delay"
+
+Formal name: <http://www.webrtc.org/experiments/rtp-hdrext/playout-delay>
+
+SDP "a= name": "playout-delay" ; this is also used in client/cloud signaling.
+
+Introduction
+
+On WebRTC, the RTP receiver continuously measures inter-packet delay and evaluates packet jitter. Besides this, an estimated delay for decode and render at the receiver is computed. The jitter buffer, the local time extrapolation and the predicted render time (based on predicted decode and render time) impact the delay on a frame before it is rendered at the receiver.
+
+This document proposes an RTP extension to enable the RTP sender to try and limit the amount of playout delay at the receiver in a certain range. A minimum and maximum delay from the sender provides guidance on the range over which the receiver can smooth out rendering. 
```

Comment:
If this is a quote from a document, please insert "Copied from <draftname>" at the top.